content/modules/bpm/partials/nav.adoc

```diff
@@ -1,25 +1,53 @@
 * xref:bpm:index.adoc[]
-** xref:bpm:getting-started.adoc[]
-** xref:bpm:admin-screens.adoc[]
-*** xref:bpm:modeler.adoc[]
-*** xref:bpm:proc-definitions.adoc[]
-*** xref:bpm:proc-instances.adoc[]
-*** xref:bpm:user-groups.adoc[]
-*** xref:bpm:user-tasks.adoc[]
+** xref:bpm:introduction.adoc[]
+** xref:bpm:whats-new.adoc[]
```

Comment:
Remove this document. What's new page is common for the whole framework and add-ons.